### PR TITLE
handle duplicate MS entries in output of "gprs list"

### PR DESCRIPTION
**Pull request by yosemitebandit**
wip..

### CODE REVIEW

**Comment by yosemitebandit** [comment]
PTAL
- handles duplicate IMSIs in the output of GPRS list by taking the larger values of both `uploaded_bytes` and `downloaded_bytes`
- various formatting adjustments
- renamed some of the test fixture files

File: openbts/tests/fixtures/gprs_list_duplicate_imsis.txt [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+         dataER:.6% (13479) recent:.5% (988) low:.8% (75) tbfER:.04% (237)
+         rrbpER:.14% (1435) recent:.14% (76) low:.3% (6) ccchER:.0% (152) recent:0% (0)
+ MS#5,TLLI=c00d1004 rrmode=PacketTransfer Multislot class: 
+                                                            Bytes:72831up/139747down Utilization=181%
+         GMM Context: imsi=901550000000542 ptmsi=0xd1004 multislot class=12 tlli=0xc00d1004 state=GmmRegisteredNormal age=95 idle=1 IPs=192.168.99.4
+         TimingError=(0.95 min=-0.88 max=2.16 avg=0.82 N=7708) RSSI=(-11 min=-46 max=-2 avg=-9.15 N=7708) CV=(13 min=8 max=30 avg=21.72 N=279) ILev=(8) RXQual=(5 min=0 max=7 avg=2.20 N=267) SigVar=(4 min=0 max=63 avg=3.67 N=268) ChCoding=(3 min=0 max=3 avg=3.00 N=3510)
+         dataER:.6% (12446) recent:.6% (2160) low:1.0% (28) tbfER:0% (192)
+         rrbpER:.11% (1131) recent:.12% (127) low:.4% (7) ccchER:.5% (2) recent:0% (0)
+ MS#7,TLLI=79a49961 rrmode=PacketTransfer Multislot class:
```
Comment (kkroo) [inline openbts/tests/fixtures/gprs_list_duplicate_imsis.txt:25]: I've never seen this before. This certainly looks like a bug to me. Do file this if you see it again

Comment (yosemitebandit) [inline openbts/tests/fixtures/gprs_list_duplicate_imsis.txt:25]: yah, @shaddi mentioned it was likely to reappear when we first added this gprs monitoring thing to openbts-python and now he found this trace, presumably from MPK.

Comment (yosemitebandit) [inline openbts/tests/fixtures/gprs_list_duplicate_imsis.txt:25]: er, sorry, were you talking about something on this line in particular or just the general "duplicate IMSI" thing?

Comment (shaddi) [inline openbts/tests/fixtures/gprs_list_duplicate_imsis.txt:25]: Yeah, this is from MPK. I agree it's _probably_ is a bug but it didn't seem to manifest in any application-layer issues when we were testing.

File: openbts/components.py [hunk 2]
```diff
@@ -541,6 +541,14 @@ def get_gprs_usage(self, target_imsi=None):
       except ValueError:
         # Casting to int failed.
         continue
+      # See if we already have an entry for the same IMSI -- we sometimes see
+      # duplicates.  If we do have an entry already, take the greater value of
```
Comment (kkroo) [inline openbts/components.py]: if all the traffic comes from the same IMSI, why not just sum the entries together?

Comment (yosemitebandit) [inline openbts/components.py]: good point, done

**Comment by kkroo** [comment]
LGTM

**Pull request merged**